app/src/main/kotlin/com/intive/picover/common/di/AppModule.kt

```diff
@@ -13,9 +13,13 @@ import kotlinx.coroutines.Dispatchers
 @InstallIn(SingletonComponent::class)
 object AppModule {
 
+	// TODO replace it using a new provider to the storage reference
```

Comment:
You will take care of this TODO? If so then add your acronym to TODO. For me this TODO is not clear what should be changed :D

Reply:
Also, I will try to clarify this comment :P

Reply:
Done :D